lib/pf/Connection/Profile.pm

```diff
@@ -609,6 +609,17 @@ sub canAccessRegistrationWhenRegistered {
     return isenabled($self->{_access_registration_when_registered});
 }
 
+=item ipskEnabled
+
+Is IPSK is enable or not on this connection profile
+
+=cut
+
+sub ipskEnabled {
```

Comment:
This doesn't seem to be used anywhere

Reply:
In the switch module (WLC and Hostapd)

Reply:
It doesn't seem it calls ipskEnabled, it just accessed the _ipsk attribute of the profile

Also for this to work as expected I think it should be "return isenabled($self->{_ipsk})"